src/payment/create-payment-strategy-registry.ts

```diff
@@ -251,9 +242,8 @@ export default function createPaymentStrategyRegistry(
             paymentMethodActionCreator,
             paymentStrategyActionCreator,
             requestSender,
-            new ChasePayScriptLoader(getScriptLoader()),
-            new WepayRiskClient(scriptLoader)
-        )
+            new ChasePayScriptLoader(scriptLoader),
+            new WepayRiskClient(scriptLoader))
```

Comment:
```suggestion
            new WepayRiskClient(scriptLoader)
        )
```

Reply:
Are we sure this extra `(` is intentional? What is it the resolution for?